### PR TITLE
release(v0.5.1): prepare release

### PR DESCRIPTION
**Pull request by Unix4ever**
This is the official v0.5.1 release.

Signed-off-by: Artem Chernyshev <artem.chernyshev@talos-systems.com>

### CODE REVIEW

File: hack/release.toml [hunk 1]
```diff
@@ -6,9 +6,9 @@ github_repo = "talos-systems/sidero"
 match_deps = "^github.com/(talos-systems/[a-zA-Z0-9-]+)$"
 
 # previous release
-previous = "v0.4.0"
+previous = "v0.5.0"
 
-pre_release = true
```
Comment (smira) [inline hack/release.toml:11]: I think we should clean up release notes here from items which were already in 0.5.0

Comment (smira) [reply hack/release.toml:11]: so it lists only _new_ items

Comment (smira) [reply hack/release.toml:11]: so no notes, that's fine as well :)

Comment (Unix4ever) [reply hack/release.toml:11]: I've added some :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,281 +1,101 @@
-## [Sidero 0.5.0](https://github.com/talos-systems/sidero/releases/tag/v0.5.0) (2022-02-14)
```
Comment (smira) [inline CHANGELOG.md:1]: CHANGLOG seems to be messed up? 0.5.0 entry removed (?)

Comment (Unix4ever) [reply CHANGELOG.md:1]: oh... yeah

**Review by smira — APPROVED** [review]

**Pull request merged**